### ISSUE TITLE
请问为什么accuracy = 0.5？

### ISSUE THREAD

**Issue by DachuanZhao** [saiwaiyanyu/bi-lstm-crf-ner-tf2.0#1]

**Comment by saiwaiyanyu** [comment]
你好，目前 tf2.0 还没有用 tf 实现 viterbi_decode 接口，代码中解码函数是 tensorflow_addons 基于python 实现的，为了提高训练效率，没有每次train-step都计算accuracy。也可以取消注释line52-line61，计算accuracy。

**Comment by DachuanZhao** [comment]
> 你好，目前 tf2.0 还没有用 tf 实现 viterbi_decode 接口，代码中解码函数是 tensorflow_addons 基于python 实现的，为了提高训练效率，没有每次train-step都计算accuracy。也可以取消注释line52-line61，计算accuracy。

看见你更新的代码了，非常感谢～！

**Comment by DachuanZhao** [comment]
> 你好，目前 tf2.0 还没有用 tf 实现 viterbi_decode 接口，代码中解码函数是 tensorflow_addons 基于python 实现的，为了提高训练效率，没有每次train-step都计算accuracy。也可以取消注释line52-line61，计算accuracy。

多问一句，```get_acc_one_step```这个函数里其实
```
        correct_prediction = tf.equal(
            tf.convert_to_tensor(tf.keras.preprocessing.sequence.pad_sequences([viterbi_path], padding='post'),
                                 dtype=tf.int32),
            tf.convert_to_tensor(tf.keras.preprocessing.sequence.pad_sequences([labels[:text_len]], padding='post'),
                                 dtype=tf.int32)
        )
```
改成
```
        correct_prediction = tf.equal(
            tf.convert_to_tensor(viterbi_path,dtype=tf.int32),
            tf.convert_to_tensor(labels[:text_len],dtype=tf.int32)
        )
```
这样就好了吧？
其实不需要padding

**Comment by saiwaiyanyu** [comment]
是的。单条样本各自计算acc，不需要 padding 。
